.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: circleci/python:3.7
         environment:
-          NEO4J_URL: bolt://localhost:7687
+          NEO4J_URL: bolt://:@localhost:7687
```

Comment:
What is that?

Reply:
Instead of URLs Neomodel uses these Database access links, which are similar to rails database links.
You can check them out here: https://apidock.com/rails/ActiveRecord/Base/establish_connection/class
Here is an example for rails from that site: "postgres://myuser:mypass@localhost/somedatabase".
Explanation is protocol://username:password@url:port
But since we have authentication disabled in CI and development I just left username and password empty.

Reply:
Ah ok I get it.